src/quacc/utils/calc.py

```diff
@@ -209,9 +209,9 @@ def run_ase_vib(
     atoms, tmpdir, job_results_dir = _calc_setup(atoms, copy_files=copy_files)
 
     # Run calculation
-    vib = Vibrations(atoms, name=os.path.join(tmpdir, "vib"), **vib_kwargs)
+    vib = Vibrations(atoms, name=Path(tmpdir, "vib"), **vib_kwargs)
     vib.run()
-    vib.summary(log=os.path.join(tmpdir, "vib_summary.log"))
+    vib.summary(log=Path(tmpdir, "vib_summary.log").as_posix())
```

Comment:
The `Vibrations()` module's `name` kwarg currently is listed as only taking in `str` as input, e.g. see [here](https://gitlab.com/ase/ase/-/blob/master/ase/vibrations/vibrations.py?ref_type=heads#L109). Same for the `log` kwarg in `summary()`. So, could you make the following changes?


```python
Vibrations(atoms, name=str(Path(tmpdir, "vib")), **vib_kwargs)
```

and then

```python
vib.summary(log=str(Path(tmpdir, "vib_summary.log"))
```